crossdock/main.go

```diff
@@ -0,0 +1,20 @@
+package main
+
+import (
+	"net/http"
+	"time"
+
+	"github.com/yarpc/yarpc-go/crossdock/client"
+	"github.com/yarpc/yarpc-go/crossdock/server"
+)
+
+func main() {
+
+	// TODO need to be able to wait till all inbounds are finished listening
+	go server.StartServerUnderTest()
+	// TODO maybe sleep?
+	time.Sleep(2 * time.Second)
```

Comment:
@abhinav here is where i need to be able to block/wait until the server is started